buildtools/gocmd/go.go

```diff
@@ -71,17 +69,14 @@ func (g *Go) ListOne(pkg string) (Package, error) {
 }
 
 // List runs `go list` to return information about packages.
-func (g *Go) List(pkgs []string) ([]Package, error) {
+func (g *Go) List(flags, pkgs []string) ([]Package, error) {
 	// Run `go list -json $PKG` and unmarshal output.
 	var output []GoListOutput
+	flags = append(flags, pkgs...)
```

Comment:
I wouldn't modify an argument, especially not a value with reference semantics (like slices). Instead, I would declare a new variable e.g. `argv := append(flags, pkgs...)`.